picard/util/tags.py

```diff
@@ -166,652 +227,1128 @@ def names(self, selector=None):
+            '"*promotional*", "*bootleg*", and "*pseudo-release*".'
+        ),
     ),
     TagVar(
         'releasetype',
         shortdesc=N_('Release Type'),
+        longdesc=N_(
+            'A multi-value tag containing the types of release assigned to the release group. See also '
+            '`%_primaryreleasetype%` and `%_secondaryreleasetype%`.'
+        ),
     ),
     TagVar(
         'remixer',
         shortdesc=N_('Remixer'),
+        longdesc=N_('The names of the remixer engineers associated with the track.'),
     ),
     TagVar(
         'replaygain_album_gain',
         shortdesc=N_('ReplayGain Album Gain'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_album_peak',
         shortdesc=N_('ReplayGain Album Peak'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_album_range',
         shortdesc=N_('ReplayGain Album Range'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_reference_loudness',
         shortdesc=N_('ReplayGain Reference Loudness'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_track_gain',
         shortdesc=N_('ReplayGain Track Gain'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_track_peak',
         shortdesc=N_('ReplayGain Track Peak'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_track_range',
         shortdesc=N_('ReplayGain Track Range'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'sample_rate',
         shortdesc=N_('File sample rate'),
+        longdesc=N_(''),
         is_file_info=True,
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        not_from_mb=True,
+    ),
+    TagVar(
+        'script',
+        shortdesc=N_('Script'),
+        longdesc=N_(
+            "The script used to write the release's track list. The possible values are taken from the "
+            'ISO 15924 standard.'
+        ),
     ),
     TagVar(
         'secondaryreleasetype',
         shortdesc=N_('FIXME:secondaryreleasetype'),
+        longdesc=N_(
+            'Zero or more secondary types (i.e.: *audiobook*, *compilation*, *dj-mix*, *interview*, '
+            '*live*, *mixtape/street*, *remix*, *soundtrack*, or *spokenword*) for the release group.'
+        ),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
-        'script',
-        shortdesc=N_('Script'),
-    ),
-    TagVar(
+        # TODO: Does this actually exist and does it come from MusicBrainz?  Is it an indicator if a track is silence?
```

Comment:
It is set by https://github.com/metabrainz/picard/blob/36014f0a6e93ab80edf3ab221e0bea5a5209770f/picard/track.py#L319-L320

According to https://github.com/metabrainz/picard/blob/36014f0a6e93ab80edf3ab221e0bea5a5209770f/picard/const/__init__.py#L100 it is when the title matches `[silence]`.

So that's set by Picard, but doesn't come directly from MusicBrainz database.